app/views/users/index.html.haml

```diff
@@ -2,19 +2,24 @@
   %h1.entry-title= t('.title')
 .post-title-divider
   %span
+= search_form_for @q do | f |
+  = f.label  :email, 'Profile Email'
+  = f.search_field :email
```

Comment:
Use the field `:email_cont`, rather than `:email`, for both the label and the search field.  This allows ransack to treat the text entry as being _contained_ within the email address - which is more useful for the user.

For instance, to find all users with email domain `spray.se`:

<img width="871" alt="screen shot 2017-09-06 at 6 09 24 pm" src="https://user-images.githubusercontent.com/9968213/30136791-980c5b6e-932e-11e7-9c09-7101185e6cfc.png">

Reply:
The search form itself should be styled similar to other search forms, e.g. companies search:

<img width="867" alt="screen shot 2017-09-06 at 6 12 58 pm" src="https://user-images.githubusercontent.com/9968213/30136927-0c03703e-932f-11e7-9378-a4b9a551a8a3.png">

In particular,
1. The label should use the (smaller) font size as shown.
2. The form should have a border (see use of Bootstrap panel styling, which provides this border)

In the current iteration - with only one field to search - we can keep the label and the field horizontally inline, and we can defer the hide/show toggling of the form.

(I will have additional comments/suggestions as this WIP effort proceeds).